pages/common/az-account.md

```diff
@@ -0,0 +1,24 @@
+# az account
+
+> Manage Azure subscription information.
+> More information: <https://docs.microsoft.com/cli/azure/account>.
+
+- Get a list of subscriptions for the logged in account:
+
+`az account list`
+
+- Set a `subscription` to be the current active subscription:
+
+`az account set --subscription {{00000000-0000-0000-0000-000000000000}}`
```

Comment:
```suggestion
`az account set --subscription {{subscription_id}}`
```

I don't know if the language is correct but that string of numbers cannot be used